components/RuiBreadcrumbs/src/RuiBreadcrumbs.css.ts

```diff
@@ -0,0 +1,88 @@
+import {css} from 'lit-element';
+
+
+/**
+	*  RuiBreadcrumbs Variables
+ 	*  Added colour to prevent empty block
+*/
+
+export const variables = css`
+	:host {
+		/* BASE STYLES */
+
+		/**
+		 * @variable The base colour for the breadcrumbs
+		 */
+		--color: #000000;
+	
+		/**
+		 * @variable Hover colour for the breadcrumbs
+		 */
+		--hover-color: #01447E;
+	
+		/**
+		 * @variable padding size
+		 */
+		--padding: 8px;
+	}
+    `;
+
+/**
+	* RuiBreadcrumbs CSS
+*/
+
+export const layout = css`
+	/**
+		* css for json array breadcrumbs
+	*/
+
+	nav.crumbs ol {
+		list-style: none;
+	}
+	
+	nav.crumbs li {
+		display: inline;
+	}
+	
+	nav.crumbs li + li::before {
+		padding: var (--padding);
+		color: var(--color);
+		content: "/\\00a0";
```

Comment:
There are certain screen readers that will pick up generated content, so it's best to use pure CSS (background-color, border etc) per the design pattern

Reply:
I would also avoid this for design reasons, what if it's a |, or a -, or a >

Reply:
I have updated this so you can decide what you enter